### PR TITLE
add getRef to the plugins methods

### PR DESCRIPTION
**Pull request by Nihaux**

### CODE REVIEW

File: HOW_TO_CREATE_A_PLUGIN.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ All functions receive an additional argument. This argument is an object contain
   getEditorState, // a function to get the current EditorState
   getReadOnly, // a function returning of the Editor is set to readOnly
   setReadOnly, // a function which allows to set the Editor to readOnly
+  getRef, // ad function to get the editor reference
```
Comment (nikgraf) [inline HOW_TO_CREATE_A_PLUGIN.md]: ad -> a

Comment (Nihaux) [inline HOW_TO_CREATE_A_PLUGIN.md]: corrected & squashed

**Comment by nikgraf** [comment]
hey @Nihaux, let's do one more change and call it `getEditorRef` instead of `getRef`? This makes it more explicit.

Can you also add a test for it? The plugins are not very much tested, but the editor is and I would like to keep up the high test coverage.

**Comment by Nihaux** [comment]
changed the name & added test.
I also had to correct a linting problem in the mention plugin (after rebasing on master)

File: draft-js-mention-plugin/src/MentionSuggestions/index.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export default class MentionSuggestions extends Component {
     suggestions: (props, propName, componentName) => {
       if (!List.isList(props[propName])) {
         return new Error(
-          `Invalid prop \`${propName}\' supplied to \`${componentName}\`. should be an instance of immutable list.`
+          `Invalid prop \`${propName}\` supplied to \`${componentName}\`. should be an instance of immutable list.`
```
Comment (nikgraf) [inline draft-js-mention-plugin/src/MentionSuggestions/index.js]: hehe, this change is a bit random in this PR, but thats fine :)

**Comment by nikgraf** [comment]
sweet, thx for your effort @Nihaux ✨

**Pull request merged**